### ISSUE TITLE
More Rail crossing signs

### ISSUE THREAD

**Issue by orwell96** [orwell96/advtrains#62]
that flashes when activated
ScottishLion mentioned moreSigns mod that has one

**Comment by gpcf** [comment]
And tracks for railroad crossings and tram tracks. :)

**Comment by mbruchert** [comment]
if you want to I could design some

**Comment by orwell96** [comment]
Gerne doch! Ich werde demnächst eh nicht dazu kommen.

**Comment by mbruchert** [comment]
welhe denn so

**Comment by orwell96** [comment]
Worauf du lust hast

**Comment by gpcf** [comment]
vielleicht lieber mal die Gleise für Bahnübergänge/Straßenbahnen. Ansonsten mal die Dinger https://commons.wikimedia.org/wiki/Category:Level_crossing_countdown_markers_in_Germany

**Comment by orwell96** [comment]
Wenn du dafür modelle machen willst wäre das cool.

**Comment by orwell96** [comment]
Vor allem Strassenbahnschienen, aber die könnten problematisch werden. Vielleicht hast du ja ne gute idee
Gleisvorlagenmodelle sind irgendwo im assets ordner

**Comment by mbruchert** [comment]
ok ich habe auch ein paar schilder gemacht :
![screenshot_20170720_145656](https://user-images.githubusercontent.com/16508248/28418428-f78d72ba-6d5b-11e7-9347-2b6ab350b526.png)
![screenshot_20170720_145708](https://user-images.githubusercontent.com/16508248/28418427-f78add52-6d5b-11e7-8d65-9e134d3f5e21.png)
![screenshot_20170720_145720](https://user-images.githubusercontent.com/16508248/28418429-f78f47c0-6d5b-11e7-9808-cf8c3fa720c4.png)
mache mich dann mal an den bahnübergang

**Comment by mbruchert** [comment]
und bitte packe die assets auf nen eigenen branch

**Comment by mbruchert** [comment]
der Bahnübergang ist fertig

**Comment by gpcf** [comment]
Können wir mal nen Screenshot vom Bahnübergang sehen? Die schilder sehen aber schon mal gut aus, fügen sich gut in den Rest des Spiels ein.

**Comment by mbruchert** [comment]
ok aber ich kann nur ein bild vom bahnübergang in blender zeigen ich habe ihn noch nicht im spiel
![untitled](https://user-images.githubusercontent.com/16508248/28420028-1d36d6c8-6d61-11e7-8a34-4aee01dff1b9.png)

**Comment by gpcf** [comment]
sieht schon mal nicht schlecht aus :)

**Comment by mbruchert** [comment]
ich habe noch ein schild gemacht
![screenshot_20170720_160419](https://user-images.githubusercontent.com/16508248/28421350-37a13af4-6d65-11e7-9aba-aca56613995f.png)

**Comment by orwell96** [comment]
Jo. Kannst du die blenderdatei hochstellen oder an orwell@bleipb.de schicken? Dann kann ich sie einbauen.

**Comment by mbruchert** [comment]
[adv_signs.zip](https://github.com/orwell96/advtrains/files/1543839/adv_signs.zip)
